### DISCUSSION TITLE
Предложение: красить :visited ссылки

### DISCUSSION THREAD

**Discussion by maxmaxme** [doka-guide/platform#926] [Идеи]
Чтобы явно видеть разделы, которые уже прочитал, предлагаю выделять их другим цветом.

Пример: https://learn.javascript.ru/

<img width="320" alt="Снимок экрана 2021-10-13 в 21 34 04" src="https://user-images.githubusercontent.com/26428493/137192784-c1c7ed9b-5d29-40d4-8660-25fe2d042015.png">

**Comment by solarrust** [comment]
Привет!
Мы обсудили этот вопрос на созвоне редакции и у нас возникли разные мнения на эту тему. 
Часть из нас поддерживает это интерфейсное решение, а у части возникли обоснованные сомнения в необходимости подкрашивать ссылки.
Было бы круто узнать реальный рабочий кейс, когда это может пригодиться. Это поможет нам продолжить диалог на эту тему.
